app/controllers/admin/payroll_runs_controller.rb

```diff
@@ -0,0 +1,30 @@
+module Admin
+  class PayrollRunsController < BaseAdminController
+    before_action :ensure_service_operator
+
+    def index
+      @payroll_runs = PayrollRun.all.order(created_at: :desc)
```

Comment:
The `.all` here is redundant as the `order` scope will return a relation that defaults to including all the records.